variables.tf

```diff
@@ -3,10 +3,11 @@ variable "traditional_default_os_user" {
   type        = "map"
 
   default = {
-    coreos = "core"
-    centos = "centos"
-    ubuntu = "ubuntu"
-    rhel   = "ec2-user"
+    coreos  = "core"
+    centos  = "centos"
+    ubuntu  = "ubuntu"
+    rhel    = "ec2-user"
+    windows = "dcosadmin"
```

Comment:
the default Windows admin user is "Administrator", we should keep that.

Reply:
The Azure Resource Manager blocks any attempt to use the username Administrator or Admin when provisioning a VM due to security reason (I was also surprised). Therefore had to restrict it with "dcosadmin", so that user will not face the issue of banned "Administrator" user

Reply:
holy ...